goc/rpm/oasis-goc.spec

```diff
@@ -59,14 +59,24 @@ This package contains files for oasis-replica.opensciencegrid.org
 
 %files replica
 /etc/cron.d/cvmfs
+/etc/init.d/oasis-replica-initclean
 /etc/squid/oasiscustomize.sh
 /etc/httpd/conf.d/cvmfs.conf
 /etc/iptables.d/60-local-cvmfs
 /etc/logrotate.d/cvmfs
 /etc/sysconfig/frontier-squid
 /var/www/html/robots.txt
+/usr/lib/systemd/system/oasis-replica-initclean.service
 %defattr(-,root,root)
 
+%post
+# restart apache and iptables if they are active
+for service in httpd iptables; do
+    if systemctl is-active --quiet $service; then
+        systemctl reload $service
```

Comment:
This doesn't actually restart the services, just reloads their configuration. This should work fine for httpd (it's an example in the man page) and presumably it works for iptables, too.

Reply:
It's true, the comment is a little misleading.